source/extensions/tracers/common/ot/opentracing_driver_impl.cc

```diff
@@ -104,6 +104,14 @@ void OpenTracingSpan::log(SystemTime timestamp, const std::string& event) {
   finish_options_.log_records.emplace_back(std::move(record));
 }
 
+void OpenTracingSpan::setBaggage(absl::string_view key, absl::string_view value) {
+    span_->SetBaggageItem(key, value);
+}
+
+std::string OpenTracingSpan::getBaggage(absl::string_view key) {
+    span_->BaggageItem(key);
```

Comment:
these are all defined in https://github.com/opentracing/opentracing-cpp/blob/be1148548d9e202e6cb28067a49fd9694c33f15b/include/opentracing/span.h#L140